app/main.development.js

```diff
@@ -874,7 +882,9 @@ app.on("ready", async () => {
     webPreferences: {
       nodeIntegration: false,
       contextIsolation: true
-    }
+    },
+    width: 1024,
+    height: 768
```

Comment:
I swear this worked in my initial test but now it's back to the tiny 800x600 window each time.  Dunno what the secret sauce is here.